services/system/AuthSys/include/services/system/AuthSys.hpp

```diff
@@ -11,6 +11,8 @@
 namespace SystemService
 {
 
+   /// The public key type used by this auth service.
+   /// It is encoded as an X.509 Subject Public Key Info.
```

Comment:
SubjectPublicKeyInfo (no spaces) is the name of the ASN.1 structure.

Reply:
Fixed